qiskit/providers/backend_compat.py

```diff
@@ -0,0 +1,249 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=invalid-name
```

Comment:
I wouldn't care if we disabled this rule globally, but out of curiosity, what's it complaining about here?

Reply:
It's actually a copy paste error (as is the module docstring). To create this file I started with `cp backend.py backend_compat.py` and then just deleted the python contents. I'll remove this:

```suggestion


```

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/8759/commits/2c8caa970f4aca3967b3cf22731b1ee04e0c8494